src/PolyhedralGeometry/triangulations.jl

```diff
@@ -1,3 +1,42 @@
+using TOPCOM_jll
+
+function topcom_regular_triangulations(pts::Union{SubObjectIterator{<:PointVector}, AbstractMatrix, Oscar.MatElem}; full::Bool=false)
+    input = homogenized_matrix(pts, 1)
+    inputstr = join(["["*join(input[i,:], ",")*"]" for i in 1:nrows(input)],",\n")
+    in = Pipe()
+    out = Pipe()
+    err = Pipe()
+    Base.link_pipe!(in, writer_supports_async=true)
+    Base.link_pipe!(out, reader_supports_async=true)
+    Base.link_pipe!(err, reader_supports_async=true)
+    cmd = Oscar.TOPCOM_jll.points2triangs()
+    if full
+        cmd = Oscar.TOPCOM_jll.points2finetriangs()
+    end
+    proc = run(pipeline(`$(cmd) --regular`, stdin=in, stdout=out, stderr=err), wait=false)
+    task = @async begin
+        write(in, "[\n$inputstr\n]\n")
+        close(in)
+    end
+    close(in.out)
+    close(out.in)
+    close(err.in)
+    result = Vector{Vector{Vector{Int}}}()
+    for line in eachline(out)
+        m = match(r"{{.*}}", line)
+        triang = replace(m.match, "{"=>"[")
+        triang = replace(triang, "}"=>"]")
+        triang = convert(Vector{Vector{Int}},JSON.parse(triang))
+        push!(result, Polymake.to_one_based_indexing(triang))
+    end
+    wait(task)
+    if !success(proc)
+        error = eof(err) ? "unknown error" : readchomp(err)
+        throw("Failed to run TOPCOM: $error")
+    end
+    return result
+end
+
```

Comment:
Very nice. This indeed leads to a huge speedup from about 2 hours to about 1 minutes (on my laptop) for the following example:

> f = convex_hull( [-1 -1 -1; 2 -1 -1; -1 -1 5])
> zero = [0 for i in 1:ambient_dim(f)]
> V = lattice_points(f)
> V = [Vector{fmpq}(v) for v in V if !iszero(v)]
> pts = vcat(matrix(QQ, transpose(zero)), matrix(QQ, transpose(hcat(V...))))
> trias = star_triangulations(pts; full=true, regular=true)

At some point (future PR, not critical at this point I believe), it might be interesting to add this as a test, if a runtime of one minute is considered ok. If yes, this should (and currently does) lead to 14295 FRSTs. Alternatively, a smaller example considers `f = convex_hull( [-1 -1 -1; 2 -1 -1; -1 2 -1])` and leads to 79 FRSTs.